Modules/_sre/sre.c

```diff
@@ -1249,19 +1250,36 @@ _sre.SRE_Pattern.sub
     /
     repl: object
     string: object
-    count: Py_ssize_t = 0
+    count: object(c_default="NULL") = 0
```

Comment:
It could be easier to implement a custom converter (see docs for `PyArg_Parse`, format unit `O&`).
```
    count: object(converter="my_converter_for_ssize_t_except_RegexFlag") = 0
```